README.md

```diff
@@ -13,21 +13,27 @@ This CloudFormation template is **NOT** intended for deployment in a production
 ## Getting Started
 
 - Make sure you have the latest version of `awscli` installed on your terminal.
-- Make sure you run this from the `us-east-1` (North Virginia) region on the AWS Console.
+- ~~Make sure you run this from the `us-east-1` (North Virginia) region on the AWS Console.~~ Run now in every region.
```

Comment:
Instead of those tildas, you can just replace the line with "You can run this in every AWS region enabled by default"